src/utils.ts

```diff
@@ -0,0 +1,20 @@
+export const atLeastHaVersion = (
```

Comment:
Please rename file and move into utils folder after changing PR target branch to dev.